docs/guides/earning-cred.md

```diff
@@ -1,5 +1,50 @@
 ---
 title: 💧 Earning Cred
+description: How to get Cred from your contributions.
```

Comment:
"how to get Cred" makes it sound like there will be actionable items, maybe rephrase to "How you get Cred from your contributions"